dotnet/src/Microsoft.Agents.AI.Workflows.Declarative/Interpreter/DeclarativeActionExecutor.cs

```diff
@@ -54,6 +54,12 @@ protected DeclarativeActionExecutor(DialogAction model, WorkflowFormulaState sta
 
     protected virtual bool EmitResultEvent => true;
 
+    /// <inheritdoc/>
+    public ValueTask ResetAsync()
```

Comment:
I wonder if we should make that a default interface implementation under `#if NET`